recipes/openapi-generator/all/conandata.yml

```diff
@@ -1,4 +1,10 @@
 sources:
+  "7.10.0":
+    url: "https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/7.10.0/openapi-generator-cli-7.10.0.jar"
+    sha256: "615e014705af34861e789e0b2a11075d3c80db134f881e937265479a4a83996e"
+  "7.9.0":
```

Comment:
thanks @tomas-krupa !

could you amend the PR to just add `7.10.0`? unless there's a strong reason where a user might want to use `7.9` when `7.10` is already available - thanks!